docs/HowTo/Store-Keys-Vaults/Use-AWS.md

```diff
@@ -0,0 +1,36 @@
+---
+description: How to store keys in AWS Secrets Manager
+---
+
+# Use Web3Signer with AWS Secrets Manager
+
+Web3Signer supports storing BLS signing keys in [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/).
+
+The [AWS Secrets Manager documentation](https://docs.aws.amazon.com/secretsmanager/latest/userguide/intro.html) provides
+the information you need to get started.
+
+## Store a private key in AWS Secrets Manager
+
+You need an AWS profile to use AWS Secrets Manager.
+
+Use the [Create a secret](https://docs.aws.amazon.com/secretsmanager/latest/userguide/manage_create-basic-secret.html)
```

Comment:
I think it's worth adding an example on how BLS private keys should be stored in the AWS Secrets Manager.
Here is a Java example:
```java
final SecretsManagerClient secretsManagerClient =
      SecretsManagerClient.builder()
          .region(Region.of(AWS_REGION))
          .build();
final String secretNamePrefix = "web3signer-aws-integration/";
final String secretName = secretNamePrefix + UUID.randomUUID();
final CreateSecretRequest secretRequest =
      CreateSecretRequest.builder().name(secretName).secretString(SECRET_VALUE).build();
secretsManagerClient.createSecret(secretRequest);
```